app/views/layouts/_header.html.slim

```diff
@@ -24,3 +22,7 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
         - else
           li
             = link_to('Login', new_user_session_path)
+- if user_signed_in?
+  .navbar-brand.topnav.breadcrumb
+    = render_breadcrumbs separator: " / "
+    
```

Comment:
I think the main idea was to get breadcrums out from the header, and now we just have 2 navbars in header, it's not good.